docs/layers/identity/centralized-terraform-access.mdx

```diff
@@ -36,21 +30,185 @@ assume the role in any account.
+
+## Using AWS Identity Center for Single Sign On
+
+Now how does AWS IAM Identity Center fit in? Let’s see how we can connect AWS Teams and Team roles with Identity Center to complete the picture.
+
+### Connecting a Permission Set to an AWS Team
+
+Identity Center has the concept of Permission Sets. Permission Sets determine what a user can or cannot do. We simply connect those Permission Sets to one of our Teams.
+
+We define a special Permission Set for the `core-identity` account, called the `Identity{X}TeamAccess` Permission Set. This Permission Set has access to assume the related AWS Team. For example, the `IdentityDeveloperTeamAccess` permission set can assume the `developers` team and the `IdentityDevopsTeamAccess` permission set can assume the `devops` team.
+
+From their AWS Team, users can apply Terraform wherever they have access
+
+
+```mermaid
+flowchart LR
+    subgraph identity_center_permission_sets["Identity Center Permission Sets"]
+        developer_access["IdentityDeveloperTeamAccess"]
+        devops_access["IdentityDevopsTeamAccess"]
+    end
+
+    subgraph core_identity_teams["core-identity Teams"]
+        developers["developers"]
+        devops["devops"]
+    end
+
+    subgraph plat_sandbox_roles["plat-sandbox Team Roles"]
+        terraform_sandbox["terraform"]
+    end
+
+    subgraph plat_prod_roles["plat-prod Team Roles"]
+        terraform_prod["terraform"]
+        planner_prod["planner"]
+    end
+
+    developer_access --> developers
+    devops_access --> devops
+
+    developers --> terraform_sandbox
+    developers --> planner_prod
+
+    devops --> terraform_sandbox
+    devops --> terraform_prod
+```
+
+### Centrally Manage Users and Groups with an IdP
+
+We still need some method to manage users and groups. AWS Teams and Team roles grant groups of users access, but do not define or manage users. AWS IAM Identity Center ties together Teams and Team roles with a predefined set of users and groups.
+
+Although you may choose to manage users and groups directly in Identity Center, the vast majority of our customers choose to use a SAML IdP as the source of truth for their users and groups. With a SAML IdP, users and groups are not created in AWS. Instead, we connect a third-party Identity Provider, such as GSuite, JumpCloud or Okta. Then we manage all Users and Groups in that IdP
+
+With “automatic provisioning” in AWS, we can synchronize all users and groups from the IdP into AWS. Then we can create Permission Sets in AWS to assign to those groups. We can assign Permission Sets to Groups for any given account. A Permission Set, such as `AdministratorAccess` or `ReadOnlyAccess`, is granted to a given group for some account. This way, we only need to define the Permission Sets once each
+
+```mermaid
+flowchart LR
+    subgraph external_idp["External Identity Provider"]
+        user1["User 1"]
+        user2["User 2"]
+        user3["User 3"]
```

Comment:
Somewhere, we need to disambiguate between IAM Users and SSO users. These are presumably not IAM users.

Reply:
added example user and group names for a better example